src/kafka/kafka_hello_world.clj

```diff
@@ -0,0 +1,92 @@
+(ns kafka.kafka-hello-world
+  (:gen-class)
+  (:import
+    (java.time Duration)
+    (java.net InetAddress)
+    (org.apache.kafka.common.serialization StringSerializer StringDeserializer)
+    (org.apache.kafka.clients.producer KafkaProducer ProducerRecord)
+    (org.apache.kafka.clients.consumer KafkaConsumer)))
+
+(defn build-producer
+  [producer-properties]
+  (KafkaProducer. producer-properties))
+
+(defn build-consumer
+  [consumer-properties topic]
+  (let [consumer        (KafkaConsumer. consumer-properties)
+        subscribe-topic (.subscribe consumer [topic])]
+    consumer))
+
+(defn new-msg
+  [topic key msg]
+  (ProducerRecord. topic key msg))
```

Comment:
Same comment as before

Reply:
I saw you commented on this, but it went away when I refresh the page, can you leave the comment again?